docs/user/openstack/customization.md

```diff
@@ -1,8 +1,22 @@
-# OpenStack Platform Customization
+# Installation customizations
+
+This document describes the ways of possible installation customizations.
+
+## Table of Contents
+
+- [Installation customizations](#installation-customizations)
+  - [OpenStack Platform Customization](#openStack-platform-customization)
```

Comment:
You left an uppercase `S` in there, it can't find the anchor.

Reply:
done :)